pipelines/internal/commands/run/run.go

```diff
@@ -587,52 +601,65 @@ func namedListOf(input, defaultPrefix string) map[string]string {
 	return output
 }
 
-func expandZones(project string, input []string) ([]string, error) {
-	var zones, prefixes []string
-	for _, zone := range input {
-		if strings.HasSuffix(zone, "*") {
-			prefixes = append(prefixes, zone[:len(zone)-1])
+func expandPrefixes(
+        project string,
+        input []string,
+        allValues func(project string, service *compute.Service) ([]string, error)) ([]string, error) {
+    var locales, prefixes []string
```

Comment:
The variables should be generic to match the function. locales -> results

Reply:
Done.